src/Ombi/ClientApp/src/app/requests-list/components/grid-spinner/grid-spinner.component.html

```
@@ -1,5 +1,12 @@
-<div class="loading-shade" *ngIf="loading">
-    <div class="row justify-content-md-center top-spacing loading-spinner">
-        <mat-spinner [color]="'accent'"></mat-spinner>
+<div class="loading-container" *ngIf="loading">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)